clinics/test_clinic.py

```diff
@@ -9,8 +9,8 @@ class TestClinic(Clinic):
         "name": "Test Location A",
         "state": "CA",
         "zip": "90210",
-        "earliest_appointment_day": "February 1 2021",
-        "latest_appointment_day": "February 28 2021",
+        "earliest_appointment_day": "March 1 2021",
+        "latest_appointment_day": "March 31 2021",
```

Comment:
Was this breaking something?

Reply:
I thought so, but I just tested with the original and it was working, so I must've been mistaken. Reverted.